src/codegen/llvm/target_platform.hpp

```diff
@@ -27,12 +27,12 @@ class Platform {
 
   private:
     /// Name of the platform.
-    const std::string name = Platform::DEFAULT_PLATFORM_NAME;
+    std::string name = Platform::DEFAULT_PLATFORM_NAME;
```

Comment:
I need to revisit this actually, the problem is in the wrapper class in `pynmodl.cpp` there I create an instance of Platform but it gets initialized by copy, which the const will not allow here. It's probably obvious to fix this, I'll check tomorrow.

Reply:
OK, checked this - think it would be best to drop the const there because otherwise you can't assign to `Platform`. That makes it a bit restrictive, for no good reason, imho.